core/sequencer/sequencer.go

```diff
@@ -59,7 +59,7 @@ var (
 )
 
 // MaxBatchSize limits the number of mutations that will be processed per epoch.
-var MaxBatchSize = 1000
+var MaxBatchSize = int32(1000)
```

Comment:
this is const, isn't it?

Reply:
The thought is that this should be a flag at some point, but we'll leave that for later.